tests/drivers/pwm/pwm_api/boards/native_sim.overlay

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024 Kickmaker
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	pwm0: pwm0 {
+		compatible = "zephyr,fake-pwm";
```

Comment:
Isn't it needed to add `DEFINE_FFF_GLOBALS;` in the test code so that this fake driver can be actually used with it?